src/app/drive/components/FileViewer/FileViewerWrapper.tsx

```diff
@@ -82,7 +82,6 @@ const FileViewerWrapper = ({
 
   useEffect(() => {
     setBlob(null);
-    navigationService.replaceState(currentFile?.uuid);
```

Comment:
Because when the file viewer was opened the url was changed again and that is already done correctly at the time of opening the file.
It was something that had been done before, I have been testing it and trying to understand why to do it twice but I have not found any reason.

Reply:
Perfect, thanks for the explanation 🚀